pkg/operator/discover/discover.go

```diff
@@ -136,6 +136,7 @@ func (d *Discover) createDiscoverDaemonSet(namespace, discoverImage, securityAcc
 								k8sutil.NamespaceEnvVar(),
 								k8sutil.NodeEnvVar(),
 								k8sutil.NameEnvVar(),
+								k8sutil.DiscoverDaemonUdevBlacklistEnvVar(),
```

Comment:
Why do we need to set Env Var here? 
It is already present in rook-operator deployment which can be referred by all the daemons without copying the data.

Reply:
I thought you can only use `envFrom` for Secrets and ConfigMaps, I can refer to ConfigMaps by moving `DISCOVER_DAEMON_UDEV_BLACKLIST` from `rook-ceph-operator` Deployment to `rook-ceph-operator-config` ConfigMap as @leseb suggested here: https://github.com/rook/rook/pull/5198#pullrequestreview-389097089